reactor-netty-core/src/test/java/reactor/netty/resources/DefaultPooledConnectionProviderTest.java

```diff
@@ -129,8 +128,8 @@ void fixedPoolTwoAcquire() throws Exception {
 		java.util.concurrent.Future<?> f1 = null;
 		java.util.concurrent.Future<?> f2 = null;
 		Future<?> sf = null;
-		try (AddressResolverGroup<?> resolver =
-				NameResolverProvider.builder().build().newNameResolverGroup(TcpResources.get(), true)) {
+		AddressResolverGroup<?> resolver = DefaultAddressResolverGroup.INSTANCE;
```

Comment:
is that on purpose? it looks like a leftover from another attempt at fixing the bug, so I just want to make sure.

Reply:
mmm tried to combine two test in one
let me create a separate test